src/system/PlatformEventSupport.h

```diff
@@ -31,6 +31,10 @@ class PlatformEventing
 public:
     static CHIP_ERROR ScheduleLambdaBridge(System::Layer & aLayer, LambdaBridge && bridge);
     static CHIP_ERROR StartTimer(System::Layer & aLayer, System::Clock::Timeout aTimeout);
+#if !CHIP_SYSTEM_CONFIG_NO_LOCKING
+    static void LockMatterStack();
+    static void UnlockMatterStack();
```

Comment:
This is no longer "PlatformEventing" or "PlatformEventSupport"; this has nothing to do with "events"....  Might be worth checking with @andy31415 about how this should be layered.

Reply:
Yes, I have created PlatformLockSupport files and moved these functions to it